frontend/src/lib/doc-util.tsx

```diff
@@ -51,16 +51,19 @@ export const DOC_LINKS = {
   MANAGE_APPLICATIONS: `${DOC_BASE_PATH}/applications/index#managing-applications`,
   ANSIBLE_JOBS: `${DOC_BASE_PATH}/clusters/index#ansible-config-cluster`,
   POLICIES_OVERVIEW: `${DOC_BASE_PATH}/governance/index#policy-overview`,
-  HYPERSHIFT_INTRO: `${DOC_BASE_PATH}/clusters/index#hosted-control-planes-intro`,
-  HYPERSHIFT_DEPLOY_AWS: `${DOC_BASE_PATH}/clusters/index#hosted-control-planes-manage-aws`,
-  HYPERSHIFT_MANAGE_KUBEVIRT: `${DOC_BASE_PATH}/clusters/index#hosted-control-planes-manage-kubevirt`,
-  HOSTED_ENABLE_FEATURE_AWS: `${DOC_BASE_PATH}/clusters/index#enable-or-disable-hosted-control-planes`,
-  HYPERSHIFT_OIDC: `${DOC_BASE_PATH}/clusters/index#hosted-create-aws-secret`,
-  GITOPS_CONFIG: `${DOC_BASE_PATH}/applications/index#gitops-overview`,
+  GITOPS_CONFIG: `${DOC_BASE_PATH}/gitops/gitops-overview`,
   DEPRECATIONS_AND_REMOVALS: `${DOC_BASE_PATH}/release_notes/release-notes#deprecations-removals`,
   ENABLE_OBSERVABILITY: `${DOC_BASE_PATH}/observability/observing-environments-intro#enabling-observability-service`,
   ACCESSING_CONSOLE: `${DOC_BASE_PATH}/web_console/web-console#accessing-your-console`,
-  ROSA_DISCOVERY_AUTOIMPORT_POLICY_EXAMPLE: `${DOC_BASE_PATH}/clusters/cluster_mce_overview#import-discover-rosa`,
+  ROSA_DISCOVERY_AUTOIMPORT_POLICY_EXAMPLE: `${DOC_BASE_PATH}/multicluster_engine_operator_with_red_hat_advanced_cluster_management/mce-acm-integration#import-discover-rosa`,
```

Comment:
We should change `multicluster_engine_operator_with_red_hat_advanced_cluster_management/mce-acm-integration` to `mce_acm_integration/acm_integrate_import_rosa`

Reply:
@dockerymick Why do you think that the path will change compared to https://content-stage.docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.12/html-single/multicluster_engine_operator_with_red_hat_advanced_cluster_management/mce-acm-integration#import-discover-rosa when the docs become available in production without the `content-stage` subdomain?

Is `content-stage.docs.redhat.com` out-of-date? If so, how often is it updated from our docs?

Reply:
nvm, I made a comment in the open PR.